cmd/task/task.go

```diff
@@ -49,6 +49,7 @@ func main() {
 			log.Print(err.Error())
 			os.Exit(err.Code())
 		}
+		log.Fatal(err)
```

Comment:
This should be `log.Print(err.Error())` IMO.

`log.Fatal()` will call `log.Print()` followed by `os.Exit(1)` - which is technically what we want. However, we should stick to exiting with the `errors.CodeUnknown` const that we have defined for clarity.

Reply:
@pd93 fixed